.github/workflows/check.yml

```diff
@@ -0,0 +1,22 @@
+name: Check
+
+on: pull_request
+
+jobs:
+  validation:
```

Comment:
I like the idea of having all validation-related jobs in one file. However, I think naming can be improved

* I think a better name for this file and the name is `validation`, as it covers more ground
* `validation` job can be renamed to `validate-wrapper` to be more precise